services/src/northbound/src/main/java/org/openkilda/northbound/service/SwitchService.java

```diff
@@ -86,4 +87,11 @@
      * @return the synchronization result.
      */
     RulesSyncResult syncRules(String switchId);
+
+    /**
+     * Removed meter from the switch.
```

Comment:
Removed -> Removes. Could you please also divide the function describing section and params with \n?